### PR TITLE
Update info on languages in README and docs

### PR DESCRIPTION
**Pull request by mazameli**
This PR updates our README and several docs pages with policy info and links re: i18n.

- README update (also cut the link to Gitter since we're not using that anymore)
- Updated admin language settings doc to clarify things
- Got rid of redundant FAQ page and updated the correct one, plus added it to the FAQ TOC. Updated list of available languages as of 0.35.
- Bunch of automatic whitespace changes thanks to Prettier

### CODE REVIEW

File: docs/faq/general/what-languages-can-be-used-with-metabase.md [hunk 1]
```diff
@@ -1,24 +1,39 @@
...
+- Norwegian Bokmål
+- Turkish
+- Ukrainian
+- Vietnamese
+
+## Policy for adding and removing translations
+
+The community contributes to Metabase translations on our [POEditor project](https://poeditor.com/join/project/ynjQmwSsGh). If you're a fan of Metabase and would like to see it made available in a language you're fluent in, we'd love your help!
+
+For new translations to be added to Metabase they must reach 100%. Once they are, we'll add them in the next major or minor release of Metabase. All _existing_ translations in Metabase _must stay at 100%_ to continue being included in the next _major_ version of Metabase. This is to ensure that no one encounters a confusing mishmash of English and another language when using Metabase.
```
Comment (dacort) [inline docs/faq/general/what-languages-can-be-used-with-metabase.md]: Minor grammatical thing here. "Once they are *complete*"? Or maybe "Once they do"?

**Review by dacort — APPROVED** [review]
One minor comment, but otherwise 👍

File: docs/faq/general/what-languages-can-be-used-with-metabase.md (hunk 1 above)
Comment (mazameli) [reply docs/faq/general/what-languages-can-be-used-with-metabase.md]: Good catch

**Comment by cenerator** [comment]
Hi Maz,
When will the Turkish version be added, any plan ? It looks like 100%

**Pull request merged**